Examples/Swift/InsetMapExample.swift

```diff
@@ -0,0 +1,57 @@
+import Mapbox
+
+@objc(InsetMapExample_Swift)
+
+class InsetMapExample_Swift: UIViewController, MGLMapViewDelegate {
+    var mapView = MGLMapView()
```

Comment:
For consistency, these map view variables should be declared as `var mapView: MGLMapView!`

Force unwrapping with `!` isn't great from a production standpoint, but I'd rather us be consistent.